### DISCUSSION TITLE
How to build libscreenpipe.dylib?

### DISCUSSION THREAD

**Discussion by clancy-sudo** [mediar-ai/screenpipe#736] [General]
I'm still learning to code. 🫣

How does libscreenpipe.dylib, libscreenpipe_x86_64.dylib, libscreenpipe_arm64.dylib work exactly? I don't see them created when I run cargo build.

How do I build them and where are they used?

**Comment by louis030195** [comment]
hey @clancy-sudo

closing this as part of our cleanup - screenpipe v2.2.0 is now out with major improvements!

check it out: https://x.com/louis030195/status/2014403576475615312

if this is still relevant or you have questions, feel free to reopen this discussion. thanks for being part of the community!
